guide/setup.md

```diff
@@ -26,11 +26,11 @@ The next screen should look like this:
 
 ![](../images/v1.0/protect_wallet.png)
 
-The App prompts you to enter a password to protect access to your new wallet to your full p2p node. Proceed to choose a password (8 or more characters) and to confirm it.
+The App prompts you to enter a password to protect the access to your new wallet to your full p2p node. Proceed by choosing a password (8 or more characters) and confirming it.
 
-> You will need to use the newly created password to access your wallet after you have locked access to the App. We recommend saving the password in a password manager such as 1Password.
+> You will need to use the newly created password to access your wallet after you have locked the access to the App. We recommend saving the password in a password manager such as 1Password.
 
-> Until you backup your wallet, you will not be able to access it without your password. There is no way to restore a wallet that wasn't backed up the password has been forgotten.
+> Until you back up your wallet, you will not be able to access it without your password. There is no way to restore a wallet that wasn't backed up if the password has been forgotten.
```

Comment:
Does that mean that after backing up one _can_ access their wallet without password?

Reply:
12 words restore doesn't require a password. user will be asked for a new password as part of the restore process. Let's clarify.